### PR TITLE
Minor log message changes (to help #6341)

### PR DESCRIPTION
**Pull request by caglar10ur**

### CODE REVIEW

**Comment by cgtexmex** [comment]
LGTM

File: lib/apiservers/portlayer/restapi/handlers/containers_handlers.go [hunk 1]
```diff
@@ -223,7 +223,7 @@ func (handler *ContainersHandlersImpl) CommitHandler(params containers.CommitPar
 	}
 
 	if err := h.Commit(context.Background(), handler.handlerCtx.Session, params.WaitTime); err != nil {
-		log.Errorf("CommitHandler error on handle(%s) for %s: %#v", h.String(), h.ExecConfig.ID, err)
+		log.Errorf("CommitHandler error on handle(%s) for %s: %s", h.String(), h.ExecConfig.ID, err)
```
Comment (vburenin) [inline lib/apiservers/portlayer/restapi/handlers/containers_handlers.go]: No need to call String for stringers. %s should do the job.

File: pkg/vsphere/vm/vm.go [hunk 2]
```diff
@@ -174,7 +174,6 @@ func (vm *VirtualMachine) WaitForExtraConfig(ctx context.Context, waitFunc func(
 	// https://www.vmware.com/support/developer/vc-sdk/visdk2xpubs/ReferenceGuide/vim.vm.ConfigInfo.html
 	err := property.Wait(ctx, p, vm.Reference(), []string{"config.extraConfig", object.PropRuntimePowerState}, waitFunc)
 	if err != nil {
```
Comment (vburenin) [inline pkg/vsphere/vm/vm.go]: I guess you would want just to return err... no need to check it for nil

File: lib/apiservers/portlayer/restapi/handlers/containers_handlers.go (hunk 1 above)
Comment (caglar10ur) [reply lib/apiservers/portlayer/restapi/handlers/containers_handlers.go]: It is already %s?

Comment (caglar10ur) [reply lib/apiservers/portlayer/restapi/handlers/containers_handlers.go]: Ahh the argument is still calling String(), will fix in a sec.

**Comment by caglar10ur** [comment]
@vburenin done

**Pull request merged**